README.md

```diff
@@ -23,31 +23,65 @@ There are three modules this project:
       It instruments the java.util.logging.Logger.log using JavaAssist.
 
 4. riddles folder \

```

Comment:
This is the only item that says "folder" - either they all should, or none of them sould